metriq_gym/cli.py

```diff
@@ -2,6 +2,47 @@
 
 import argparse
 
+from tabulate import tabulate
+
+from metriq_gym.job_manager import JobManager, MetriqGymJob
+from metriq_gym.provider import ProviderType
+
+LIST_JOBS_HEADERS = ["ID", "Provider", "Device", "Type", "Dispatch time (UTC)"]
+TIMESTAMP_FORMAT = "%Y-%m-%d %H:%M:%S"
+
+
+def list_jobs(args: argparse.Namespace, job_manager: JobManager) -> int:
+    """List jobs recorded in the job manager.
+
+    Args:
+        args: Parsed arguments.
+        job_manager: Job manager instance.
+    Returns:
+        Return code.
+    """
+    # Retrieve all jobs from JobManager.
+    jobs: list[MetriqGymJob] = job_manager.get_jobs()
+
+    # Display jobs in a tabular format.
+    if not jobs:
+        print("No jobs found.")
+        return 0
+
+    # Prepare data for tabulation.
+    table = [
+        [
+            job.id,
+            job.provider_name,
+            job.device_name,
+            job.job_type,
+            job.dispatch_time.strftime(TIMESTAMP_FORMAT),
+        ]
+        for job in jobs
+    ]
+    # Print the table.
```

Comment:
Seems like a very [WET](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself#WET) comment. Should remove.